src/events/types/weather/LightningStrikeEvent.ts

```diff
@@ -0,0 +1,53 @@
+import { Cancellable } from "../../Cancellable";
+import { LightningStrikeEventCause } from "./LightningStrikeEventCause";
+import { WeatherEvent } from "./WeatherEvent";
+
+/**
+ * Stores data for lightning striking
+ */
+export class LightningStrikeEvent extends WeatherEvent implements Cancellable {
+    
+    public static getBukkitClasspath(): string {
+        return 'org.bukkit.event.weather.LightningStrikeEvent';
+    }
+   
+    /**
+    * Gets the cause of this lightning strike
```

Comment:
Almost all of your method documentation is missing one space, the `*`s should all line up. I guess it's not that important, but would be nice to fix if you have the time